src/components/Editor/DebugLine.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import { Component } from "react";
+import type { SourceEditor } from "devtools-source-editor";
+import type { Frame } from "../../types";
+
+type DebugLineProps = {
+  editor: SourceEditor,
+  frame: Frame,
+  visibleSourceId: string
```

Comment:
I like the term visible, but I think `selectedSourceId` might be more consistent in the short term